include/zephyr/zbus/zbus.h

```diff
@@ -18,6 +18,8 @@ extern "C" {
 /**
  * @brief Zbus API
  * @defgroup zbus_apis Zbus APIs
+ * @since 4.2.0
```

Comment:
Not sure if this should instead be the version of the introduction of the API vs the version of its first versioning. @kartben?

Reply:
Zbus was introduced in Zephyr v3.3.0. This "since" was not clear to me.

Reply:
Let's discuss it in the Architecture WG meeting.

Reply:
Done.